src/nebula/Queue/IJobStepSource.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using ComposerCore.Attributes;
+
+namespace Nebula.Queue
+{
+    public interface IJobStepSource
```

Comment:
I wrote my naming change proposals on the "SampleJobQueue" class. Please check that out.